### PR TITLE
MEPTS-1433 | Resumo Mensal - HIV Diff FGH and CCS DB

### PR DESCRIPTION
**Pull request by TinoMuianga**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java [hunk 1]
```diff
@@ -576,7 +576,7 @@ public CohortDefinition getPatientsWhoHadTbDiagnosticAfterPreArt() {
             + "                          AND e.voided = 0 "
             + "                          AND o.voided = 0 "
             + "                          AND e.encounter_type = ${6} "
-            + "                          AND e.encounter_datetime < DATE_SUB(:endDate, INTERVAL 1 MONTH) "
+            + "                          AND e.encounter_datetime <= DATE_SUB(:endDate, INTERVAL 1 MONTH) "
```
Comment (helderjosue) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: @TinoMuianga Please add javadocs for this method

Comment (TinoMuianga) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java:590]: Done

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java [hunk 2]
```diff
@@ -513,6 +513,17 @@ public CohortDefinition getPatientsWhoStartedTptAfterPreArt() {
...
+   * <b>excluindo os utentes</b>
+   *
+   * <ul>
+   *   <li>com o registo de "Diagnóstico TB Activa (S/ N)" (Coluna 10) = "S" numa consulta clínica
+   *       (Ficha Clínica) que ocorreu no período anterior do relatório e após o início Pré-TARV
+   *       (“Data Consulta” >= “Data Início Pre-TARV” e < “Data Início do Relatório”,
+   * </ul>
+   *
+   * @return
+   */
```
Comment (helderjosue) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: This docs mismatch the definition implementation, make sure you are using the correct requirements. If yes, make the necessary changes to match the specs.

**Review by helderjosue — APPROVED** [review]
LGTM

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**